projects/demo/src/modules/components/data-list-wrapper/data-list-wrapper.template.html

```diff
@@ -0,0 +1,110 @@
+<tui-doc-page
+    header="DataListWrapper"
+    package="KIT"
+    type="components"
+>
+    <ng-template pageTab>
+        <p i18n>
+            <code>DataListWrapper</code>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Asperiores aspernatur dolorum esse fugiat fugit
```

Comment:
What's the description for this page?

Reply:
`DataListWrapper` is an abstraction over `DataList` to simplify usage in common cases where precise control is not necessary.